package-lock.json

```diff
@@ -0,0 +1,530 @@
+{
```

Comment:
Please delete `package-lock.json`, we use `yarn.lock`

Reply:
This is likely the issue you are running into, because you are using a different version of the types. If you use `yarn install` you will not have this error.

Reply:
Ah yeah, switching to node 12 and using yarn used this for me. Thanks. I will close this PR.